tests/base_test_class.py

```diff
@@ -321,7 +325,7 @@ def assertNoConsoleErrors(self):
 
             The addition of the trigger exception is due to the Report Builder tests.
             """
-            accepted_javascript_messages = r'(zoom\-in\.cur.*)404\ \(Not\ Found\)|Uncaught TypeError: Cannot read properties of null \(reading \'trigger\'\)'
+            accepted_javascript_messages = r'(zoom\-in\.cur.*)404\ \(Not\ Found\)|Uncaught TypeError: Cannot read properties of null \(reading \'trigger\'\)|Uncaught TypeError: Cannot read properties of null \(reading \'innerHTML\'\)'
```

Comment:
should we add a comment to explain where this new "ignore exception" is coming from?

Reply:
Yes we should 😃, it's done with https://github.com/DefectDojo/django-DefectDojo/pull/5148/commits/01e854a9105e4d679911cc06aaa0cf2898108256